MzML/MzmlMethods.cs

```diff
@@ -1155,7 +1155,7 @@ public static SHA1 GetSHA1Hash(string filePath)
                 using (StreamReader reader = new StreamReader(stream, bufferSize: 4096))
                 {
                     string line;
-                    byte[] buffer = new byte[1000000];
+                    byte[] buffer = new byte[2000000];
```

Comment:
These can be up to 2 billion, if we want to further guard against future crashes. I'm sure there's a trade-off on RAM, though.

Reply:
i guess these aren't too big. size increased from 1MB to 2 MB. Maybe we can just go to 10 Mb. Seems like the array is just re-used so that's not a big deal i guess.